### ISSUE TITLE
Screenreader fails in antd because of rc-select

### ISSUE THREAD

**Issue by filipjnc** [react-component/select#423]
On the first glance into the code, this library seems to follow the aria guidelines for accessibility and screen reader usage. However, when actually using a screenreader like JAWS or NVDA, it fails to read the options inside the listbox and some other small issues.
